lib/ansible/plugins/callback/foreman.py

```diff
@@ -80,154 +80,169 @@
 from ansible.plugins.callback import CallbackBase
 
 
+def build_log(data):
+    for source, msg in data:
+        if 'failed' in msg:
+            level = 'err'
+        elif 'changed' in msg and msg['changed']:
+            level = 'notice'
+        else:
+            level = 'info'
+
+        yield {
+            "log": {
+                'sources': {
+                    'source': source,
+                },
+                'messages': {
+                    'message': json.dumps(msg),
+                },
+                'level': level,
+            }
+        }
+
+
+def get_time():
+    """
+    Return the time for measuring duration. Prefers monotonic time but falls back to the regular
```

Comment:
The first line of docstring is a title and should be one sentence fitting 70 chars as per PEP 257